docs/config-fcos-v1_4-exp.md

```diff
@@ -201,6 +201,9 @@ The Fedora CoreOS configuration is a YAML document conforming to the following s
     * **_threshold_** (int): sets the minimum number of pieces required to decrypt the device.
   * **_mirror_** (object): describes mirroring of the boot disk for fault tolerance.
     * **_devices_** (list of strings): the list of whole-disk devices (not partitions) to include in the disk array, referenced by their absolute path. At least two devices must be specified.
+* **_kernel_arguments_** (object): describes the desired kernel arguments.
```

Comment:
Nit: this is Ignition syntax, so let's put it before the sections that are Butane sugar (`boot_device`, and in `openshift` specs, `openshift`).

Reply:
Yup.